### ISSUE TITLE
My godness! No any response any more!

### ISSUE THREAD

**Issue by huangzhenyu** [seemethere/nba_py#123]
Today early the code run normally. But become no response any more! I wait for several hours! No any repsonse!

**Comment by huangzhenyu** [comment]
Really interesting！I change another HEADER , and it could be worked normally again! Maybe the default have been used by large people all round the world, thus it is blocked by the NBA.com!

**Comment by GeorgePaul24** [comment]
What change did you do? Same thing for me

**Comment by s4njee** [comment]
Having the same issue here

**Comment by s4njee** [comment]
confirms it works when you change the user-agent header in __init__.py

**Comment by ilakisp** [comment]
i'm a big noob in this, but could you comment how you managed to fix it? I tried changing user agent headers but with no luck.

**Comment by s4njee** [comment]
I cloned the git repo, changed the line for user_agent in __init__.py to 'customagent', and then installed the package by doing pip install . inside the git folder with the changed init.py

That's pretty much it.

**Comment by jrogers239** [comment]
doesn't fix for me when i change the HEADERS variable to 'customagent'

**Comment by s4njee** [comment]
It looks like this?

`HEADERS = {
    'user-agent': ('customheader'),  # noqa: E501
    'Dnt': ('1'),
    'Accept-Encoding': ('gzip, deflate, sdch'),
    'Accept-Language': ('en'),
    'origin': ('http://stats.nba.com')
    }`

That's literally the only change I made. Make sure you install the package after you edit it too...

**Comment by jrogers239** [comment]
thanks! realized part of my problem was working with an older version of the init file, updated to most recent version and fixed the agent. all working again :) thanks for your help!

**Comment by SpecCRA** [comment]
Will we ever see a fix by @seemethere ?

**Comment by rneu31** [comment]
I've tried contacting @seemethere on here and via Twitter with no response. If nothing changes shortly before the NBA season starts back up I will fork it and handle a good amount of these issues. I'm interested in becoming a shared owner of this project so all of that isn't required but we'll see..

**Comment by bttmly** [comment]
@rneu31 if you fork this project let me know. I maintain a JS client very similar to this one. I had some ideas about trying to standardize clients across languages using some sort of shared JSON configuration – it'd make maintenance much easier – and I began that work here: https://github.com/bttmly/nba-client-template

We could consider creating a github organization to manage NBA client library projects.

**Comment by sw4r** [comment]
@rneu31  and @bttmly 

Please let me know if you need any help. I would love to contribute more.

**Comment by huangzhenyu** [comment]
@GeorgePaul24  Yeah , I change the Header to another one and delete *.pyc files. And it work normally.
But now I write a loop to scrape all the player's shot detail type stats, it comes out again.
Perhaps , time out and no response. When I rerun the code,it start again, but stop at sometime.  I try to have a idea to retry the code by itself.

**Comment by swar** [comment]
@bttmly I pulled as many endpoints as I could find from various NBA API clients as well as the stats.nba.com website. I wanted to have good starting list of endpoints to run my analysis script. Thankfully, this process made creating the documentation and API client much simpler. Similar to what you are accomplishing in nba-client-template. I wanted to ensure that this works so that I can regularly update the API client during the NBA Season or whenever new endpoints are discovered.


I completely agree with having a separate repo with markdown and a json file. Not sure on how the best way to proceed would be. I am open to hearing any thoughts you might have on this as we could potentially use most of the parameter and endpoint documentation I created in nba_api.

**Comment by martendt** [comment]
Sorry I'm late to the party here, but I've been having the same issue as above. The change to 'user-agent' worked to an extent - it stopped the endless running of the code - but now I get a '400 client error' that stats.nba.com/stats does not exist. And indeed, it doesn't, when I try to go in my browser. Is this package too outdated now?

**Comment by martendt** [comment]
Aaaand it was a syntax error. I was requesting '2018-2019' instead of '2018-19'. I'll show myself out now.

**Comment by whichen** [comment]
I tested by using "print(nba_py.Scoreboard(month=2, day=21, year=2015))"
But get error: ConnectionError: ('Connection aborted.', OSError("(60, 'ETIMEDOUT')",))
Is this package still working?

**Comment by liaoyongming** [comment]
> It looks like this?
> 
> `HEADERS = { 'user-agent': ('customheader'), # noqa: E501 'Dnt': ('1'), 'Accept-Encoding': ('gzip, deflate, sdch'), 'Accept-Language': ('en'), 'origin': ('http://stats.nba.com') }`
> 
> That's literally the only change I made. Make sure you install the package after you edit it too...

谢谢
